core/LogDeleter.php

```diff
@@ -35,33 +42,15 @@ public function __construct(RawLogDao $rawLogDao)
      */
     public function deleteVisits($visitIds)
     {
-        $this->deleteConversions($visitIds);
-        $this->rawLogDao->deleteVisitActionsForVisits($visitIds);
-
-        return $this->rawLogDao->deleteVisits($visitIds);
-    }
+        $numDeletedVisits = 0;
 
-    /**
-     * Deletes conversions by visit ID. This method cascades, so conversion items are also deleted.
-     *
-     * @param int[] $visitIds The list of visits to delete conversions for.
-     * @return int The number rows deleted.
-     */
-    public function deleteConversions($visitIds)
-    {
-        $this->deleteConversionItems($visitIds);
-        return $this->rawLogDao->deleteConversions($visitIds);
-    }
+        foreach ($this->logTablesProvider->getAllLogTables() as $logTable) {
+            if ($logTable->getColumnToJoinOnIdVisit()) {
+                $numDeletedVisits += $this->rawLogDao->deleteFromLogTable($logTable->getName(), $visitIds);
+            }
+        }
 
-    /**
-     * Deletes conversion items by visit ID.
-     *
-     * @param int[] $visitIds The list of visits to delete conversions for.
-     * @return int The number rows deleted.
-     */
-    public function deleteConversionItems($visitIds)
-    {
-        return $this->rawLogDao->deleteConversionItems($visitIds);
+        return $numDeletedVisits;
```

Comment:
before we returned only the number of visits deleted in log_visit but now we return the sum of deleted visits. I had a look and the value returned here seems to be actually not used (not even in the output of the console command anymore) so it shouldn't change anything